### PR TITLE
7/reusable card component

### PR DESCRIPTION
**Pull request by jobkarani**
#28 feat: Creates a reusable and responsive card 

elewaweb
elewamobile

### CODE REVIEW

File: apps/elewa-group-website/src/app/app.module.ts [hunk 1]
```diff
@@ -6,12 +6,14 @@ import { ScullyLibModule } from '@scullyio/ng-lib';
 import { AppComponent } from './app.component';
 
 import { HomePageModule } from '@elewa-group/pages/elewa/home'
+import {CardsModule} from 'libs/features/components/cards/src/lib/cards/cards.module'
```
Comment (IanOdhiambo9) [inline apps/elewa-group-website/src/app/app.module.ts:9]: Use domain imports i.e 
import { CardsModule } from '@elewa-group/features/components/cards'

File: libs/features/components/cards/src/lib/cards/elewa-group-card/elewa-group-card.component.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
+.card{
+    padding: 20px 30px;
+    background: #f4f4f4;
```
Comment (IanOdhiambo9) [inline libs/features/components/cards/src/lib/cards/elewa-group-card/elewa-group-card.component.scss:3]: Use variables in _variables.scss for colors, fonts etc

File: libs/features/components/cards/src/lib/features-components-cards.module.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@NgModule({
+  imports: [CommonModule],
+})
+export class FeaturesComponentsCardsModule {}
```
Comment (IanOdhiambo9) [inline libs/features/components/cards/src/lib/features-components-cards.module.ts:7]: Rename to CardsModule

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

**Comment by IanOdhiambo9** [comment]
Use fonts in _variables.scss

**Comment by IanOdhiambo9** [comment]
Resolve conflict on styles.scss

File: apps/elewa-group-website/src/app/app.module.ts (hunk 1 above)
Comment (jobkarani) [reply apps/elewa-group-website/src/app/app.module.ts]: Resolved the issue. Waiting for another review.

File: libs/features/components/cards/src/lib/cards/elewa-group-card/elewa-group-card.component.scss (hunk 2 above)
Comment (jobkarani) [reply libs/features/components/cards/src/lib/cards/elewa-group-card/elewa-group-card.component.scss]: Resolved the issue. Waiting for another review.

File: libs/features/components/cards/src/lib/features-components-cards.module.ts (hunk 3 above)
Comment (jobkarani) [reply libs/features/components/cards/src/lib/features-components-cards.module.ts]: Resolved the issue. Waiting for another review.

**Review by IanOdhiambo9 — APPROVED** [review]

**Comment by jrosseel** [comment]
Closed bounty as WON with 3 bounty points scored

**Pull request merged**